tasks/coverage/src/runtime/mod.rs

```diff
@@ -182,7 +201,7 @@ async fn request_run_code(json: impl serde::Serialize + Send + 'static) -> Resul
     tokio::spawn(async move {
         agent()
             .post("http://localhost:32055/run")
-            .timeout(Duration::from_secs(10))
+            .timeout(Duration::from_secs(50))
```

Comment:
This seems like an unrelated change? Why do we need to timeout for so long?

This goes against the philosophy of this project, we should strive to keep all the waiting times down.

Reply:
Here has many timeout at my local environment, it is valid to avoid timeout.

Reply:
We need to fix the timeout if this is the case. cc @Dunqing if you happen to know why it's timing out.

Reply:
Could you also help me update the test snapshot file? Thanks..

Reply:
Try `just submodules`. You may need to delete their directories.